README.md

```diff
@@ -20,6 +20,7 @@ ServiceFoundation enables you to create Web Services containing:
 * Support service warm-up through state customization
 * Standardized metrics (defaults to go-metrics)
 * Standardized log messages in JSON format
+* Adding route-specific meta fields to log messages
```

Comment:
could you add a description and example of UsePublicRootHandler, please?

Reply:
I added an example to the readme. The description will come later, when I update the documentation exposed to GoDocs, because that's not really helpful at the moment.